### PR TITLE
fix: secureRoute is a value not a function

### PR DESCRIPTION
**Pull request by wenzowski**
continues #544 to resolve [failed deploy](https://github.com/bcgov/CONN-CCBC-portal/runs/8083900978)

### CODE REVIEW

File: app/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,11 @@ RUN sed -i -nr '/node|yarn/p' ${APP_HOME}/.tool-versions && \
   cat ${APP_HOME}/.tool-versions | cut -f 1 -d ' ' | xargs -n 1 asdf plugin-add && \
   asdf plugin-update --all && \
   asdf install && \
-  asdf reshim
+  asdf reshim && \
+  pushd ${APP_HOME}/.asdf/installs/nodejs/$(awk '/^nodejs/ { print $2 }' .tool-versions)/lib && \
+  npm i npm corepack && \
+  rm -f package.json package-lock.json && \
+  popd
```
Comment (matthieu-foucault) [inline app/Dockerfile:48]: Is this intentionally in this PR?

**Review by matthieu-foucault — APPROVED** [review]

File: app/Dockerfile (hunk 1 above)
Comment (wenzowski) [reply app/Dockerfile:48]: yes I rebased it on the main I already merged but haven't pushed yet because I'm waiting for happo to pass (don't judge)

Comment (wenzowski) [reply app/Dockerfile:48]: it'll disappear on its own

Comment (wenzowski) [reply app/Dockerfile:48]: now it's gone 🪄

Comment (matthieu-foucault) [reply app/Dockerfile:48]: magic

**Pull request merged**